pycronofy/tests/test_client.py

```diff
@@ -152,6 +156,52 @@ def test_userinfo(client):
     assert userinfo['cronofy.type'] == 'userinfo'
 
 
+@responses.activate
+def test_create_calendar_with_duplicate_no_issues(client):
+    responses.add(
+        responses.POST,
+        url='%s/%s/calendars' % (settings.API_BASE_URL, settings.API_VERSION),
+        body="""
+        {"calendar": {"provider_name": "test_provider",
+        "profile_id": "%s",
+        "profile_name": "my-test-profile-name",
+        "calendar_id": "test-calendar-id",
+        "calendar_name": "%s",
+        "calendar_readonly": false,
+        "calendar_deleted": false,
+        "calendar_primary": false,
+        "permission_level": "unrestricted"
+        }}""" % (TEST_PROFILE_ID, TEST_CALENDAR_NAME),
+        status=200,
```

Comment:
I may be missing something but this doesn't look like it is exercising the "ignore duplicate" path that the test name suggests.

I would expect this call to have a response status of 422, and would probably need to add a response for `GET /v1/calendars` based on my (non-Python dev!) read of the `except` block.

Reply:
yep, i totally meant to make a test for that case. all done.